SparkJoinOnGPU.ipynb

```
@@ -0,0 +1,83 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Download 10.2 CUDA Driver from [here](https://developer.nvidia.com/cuda-toolkit-archive).\n",
```

Ruff (0.16.0)

[error] Undefined name `sql`

(F821)